src/com/rw/legion/DefaultMapper.java

```diff
@@ -174,17 +174,26 @@ private void tryOutput(OutputTable outputTable, LegionRecord value,
                 dataToWrite[i]
                     = StringEscapeUtils.escapeCsv(value.getData(keyList[i]));
             } else {
+                dataToWrite = new String[4];
+                dataToWrite[0] = value.getData("file_name");
+                dataToWrite[1] = value.getData("file_position");
+                dataToWrite[2] = column.getKey();
+                dataToWrite[3] = column.getValidationReason();
+                        
                 validates = false;
                 break;
             }
             
             i++;
         }
         
+        outputLine.set(StringUtils.join(dataToWrite, ","));
+        
         if (validates) {
-            outputLine.set(StringUtils.join(dataToWrite, ","));
-            outputWriters.write(outputTable.getTitle(),
-                                nothing, outputLine, outputTable.getTitle());
+            outputWriters.write(outputTable.getTitle(), nothing, outputLine,
+                    outputTable.getTitle());
+        } else {
+            outputWriters.write("skipped", nothing, outputLine, "skipped");
```

Comment:
from where does `nothing` come? I'm not saying anything is wrong with it, i just can't find where it's defined.